src/map/utils/synthutils.cpp

```diff
@@ -546,19 +538,35 @@ namespace synthutils
                 penalty += 1;
             }
 
-            if (PChar->CraftContainer->getQuantity(0) == SYNTHESIS_FAIL) // If synth breaks, lower skill up rate
-            {
-                penalty += 1;
-            }
+            // No proof this is a thing. Until proven otherwise removing.
+            // if (PChar->CraftContainer->getQuantity(0) == SYNTHESIS_FAIL) // If synth breaks, lower skill up rate
+            // {
+            //     penalty += 1;
+            // }
 
-            skillUpChance = skillUpChance / penalty; // Lower skill up chance if synth breaks
+            // https://www.bluegartr.com/threads/57123-Before-you-ask-a-stupid-crafting-question-read-this!?p=1987222&viewfull=1#post1987222
+            skillUpChance = skillUpChance / penalty; // Lower skill up chance if desyntth
 
             // Section 3: Calculate Skill Up and Skill Up Amount
             double random = xirand::GetRandomNumber(1.);
 
             if (random < skillUpChance) // If character skills up
             {
                 uint8 skillUpAmount = 1;
+                uint8 maxSkillUp    = 4; // Max skill is 0.4
+
+                if (charSkill >= 600)
+                {
+                    maxSkillUp = 1;
+                }
+                else if (baseDiff >= 6)
+                {
+                    maxSkillUp = 3;
+                }
+                else if (baseDiff >= 12)
+                {
+                    maxSkillUp = 4;
+                }
```

Comment:
```suggestion
                uint8 maxSkillUp    = 1; // Max skillup is 0.1 above level 60

                if (charSkill < 600)
                {
                    maxSkillUp = 2;
                    if (baseDiff >= 6)
                    {
                        maxSkillUp = 3;
                    }
                    if (baseDiff >= 12)
                    {
                        maxSkillUp = 4;
                    }
                }
```
I think this is what @archFF was saying (in suggestion form)? Correct me if I'm wrong.

Reply:
I have it as this atm:
![image](https://user-images.githubusercontent.com/105882754/203673446-b9a43c79-3480-46fe-84c1-3a8631380d26.png)

Reply:
Can also just put it in the if statement below:

![image](https://user-images.githubusercontent.com/105882754/203675510-bc056a7b-08b5-4da4-8ffc-b9be61b07ae6.png)